bash_completion.sh

```diff
@@ -158,7 +158,7 @@ _buildtest ()
       esac
       ;;
     config|cg)
-      local cmds="-h --help compilers edit executors validate view systems"
+      local cmds="-h --help compilers edit-test executors validate view systems"
```

Comment:
This is the section for `buildtest config` and we have command `buildtest config edit` which is a valid command. Please revert change